airbyte-integrations/connectors/source-python-http-tutorial/README.md

```diff
@@ -47,10 +47,10 @@ and place them into `secrets/config.json`.
 
 ### Locally running the connector
 ```
-python main_dev.py spec
-python main_dev.py check --config secrets/config.json
-python main_dev.py discover --config secrets/config.json
-python main_dev.py read --config secrets/config.json --catalog sample_files/configured_catalog.json
+python main.py spec
+python main.py check --config sample_files/config.json
```

Comment:
Moved this to `sample_files` since the tutorial instructs users to place config in `sample_files` and all the existing test related objects are currently there.